### PR TITLE
fix: [CDS-53380]: Adding logs for duplicate sweepingOutput exception.

### PR DESCRIPTION
**Pull request by boopesh007**
## Describe your changes
Ignoring duplicateKey exception in ECS state helper

Log line 

11:13:25,891 [main-app-pool-6] INFO  software.wings.sm.StateMachineExecutor - Execution is processing for workflowExecutionId: pzrqcxfVR-irRmMEYwF4EA, stateExecutionId: OnUmeww1QaW5rjQz-V7wQA, appId: 7Fp8EuMOSI626Km_D1gaxw, accountId: kmpySmUISimoRrJL6NL73w 

## Checklist
- [x] I've documented the changes in the PR description.
- [x] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>

You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger utAll`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/44286)

### CODE REVIEW

File: 400-rest/src/main/java/software/wings/sm/states/EcsStateHelper.java [hunk 1]
```diff
@@ -971,15 +971,19 @@ public ExecutionResponse handleDelegateResponseForEcsDeploy(ExecutionContext con
...
+                .name(context.appendStateExecutionId(InstanceInfoVariables.SWEEPING_OUTPUT_NAME))
+                .value(InstanceInfoVariables.builder()
+                        .instanceElements(allInstanceElements)
+                        .instanceDetails(instanceDetails)
+                        .skipVerification(skipVerification)
+                        .build())
+                .build());
+      }
+      catch (InvalidRequestException e){
+        log.error("Facing a duplicate key exception while saving sweepingOutputContext, ignoring", e);
```
Comment (PragyeshMishra01) [inline 400-rest/src/main/java/software/wings/sm/states/EcsStateHelper.java:985]: here, does this InvalidRequestException comes only incase of duplicate key?

Comment (abhinav-harness) [reply 400-rest/src/main/java/software/wings/sm/states/EcsStateHelper.java:985]: why cant we catch a execption?

**Review by abhinav-harness — CHANGES_REQUESTED** [review]

**Review by sainath-b — APPROVED** [review]
LGTM. @boopesh007 Make sure you test this locally and see if any NPE occurs.

**Review by sainath-b — DISMISSED** [review]
> Reviewed 3 of 3 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/44286)* status: 3 file(s) to review. 1 change requests. 1. unresolved discussions. 1 of 2 approvals obtained.

**Comment by boopesh007** [comment]
*[`400-rest/src/main/java/software/wings/sm/states/EcsStateHelper.java` line 985 at r1](https://reviewable.io/reviews/harness/harness-core/44286#-NOyN2C1txUxX8N9jl60-r1-985:-NPGKs_X4f7oDuwyALjf:bdygk4c) ([raw file](https://github.com/harness/harness-core/blob/4e6273248a0b14b78f3010fa1c6969212beef4a5/400-rest/src/main/java/software/wings/sm/states/EcsStateHelper.java#L985)):*
<details><summary><i>Previously, abhinav-harness (Abhinav Singh) wrote…</i></summary><blockquote>

why cant we catch a execption?
</blockquote></details>

No valid anymore since additional logs are only being added.

**Comment by boopesh007** [comment]
trigger messagecheck

**Review by inderschera — APPROVED** [review]
> Reviewed 1 of 3 files at r2, 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/44286)* status: 2 file(s) to review. 1 change requests. 1. unresolved discussions. 1 of 2 approvals obtained.

**Review by sainath-b — APPROVED** [review]

**Review by sainath-b — APPROVED** [review]
> Reviewed 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/44286)* status: 1 change requests. 1. unresolved discussions.

**Review by sainath-b — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/44286)* status: 1 change requests. 1. unresolved discussions.

**Review by PragyeshMishra01 — COMMENTED** [review]
> Reviewed 1 of 3 files at r2, 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/44286)* status: 1 change requests. 1. unresolved discussions.

**Review by PragyeshMishra01 — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/44286)* status: :shipit: complete! ✓

**Pull request merged**